src/encord_active/lib/db/connection.py

```diff
@@ -1,13 +1,23 @@
 import sqlite3
-from pathlib import Path
 from typing import Optional
 
-from encord_active.lib.db.__autogen__.cli.prisma import run
-from encord_active.lib.db.__autogen__.client import Prisma
-from encord_active.lib.db.__autogen__.types import DatasourceOverride
-from encord_active.lib.file_structure.base import BaseProjectFileStructure
+from encord_active.lib.db.prisma_init import generate_prisma_client
+
+try:
```

Comment:
should we move the try/except to `prisma_init` module so after calling that module we can ensure that prisma effectively works?